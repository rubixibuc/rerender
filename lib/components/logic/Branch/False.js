import React from "react";
import PropTypes from "prop-types";

const render = ({ children }) => <React.Fragment>{children}</React.Fragment>;

render.propTypes = {};

export default render;
