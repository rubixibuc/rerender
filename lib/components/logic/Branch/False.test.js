import React from "react";
import renderer from "react-test-renderer";

import False from "./True";

test("should render children when children is undefined", () => {
  const component = renderer.create(
    <False>
      <div />
    </False>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("should render children when children is null", () => {
  const component = renderer.create(
    <False>
      <div />
    </False>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("should render children when children are defined", () => {
  const component = renderer.create(
    <False>
      <div />
    </False>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
