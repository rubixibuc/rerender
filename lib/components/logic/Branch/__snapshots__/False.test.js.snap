// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render children when children are defined 1`] = `<div />`;

exports[`should render children when children is null 1`] = `<div />`;

exports[`should render children when children is undefined 1`] = `<div />`;
